src/docker.rs

```diff
@@ -14,41 +12,28 @@ use crate::extensions::CommandExt;
 use crate::id;
 
 const DOCKER_IMAGES: &[&str] = &include!(concat!(env!("OUT_DIR"), "/docker-images.rs"));
-
-lazy_static! {
-    /// Retrieve the Docker Daemon version.
-    ///
-    /// # Panics
-    /// Panics if the version cannot be retrieved or parsed
-    static ref DOCKER_VERSION: Version = {
-        let version_string = Command::new("docker")
-                                .arg("version")
-                                .arg("--format={{.Server.APIVersion}}")
-                                .run_and_get_stdout(false)
-                                .expect("Unable to obtain Docker version");
-        // API versions don't have "patch" version
-        Version::parse(&format!("{}.0", version_string.trim()))
-            .expect("Cannot parse Docker engine version")
-    };
-
-    /// Version requirements for user namespace.
-    ///
-    /// # Panics
-    /// Panics if the parsing fails
-    static ref USERNS_REQUIREMENT: VersionReq = {
-        VersionReq::parse(">= 1.24")
-            .expect("Unable to parse version requirements")
-    };
+const DOCKER: &str = "docker";
+const PODMAN: &str = "podman";
+
+fn get_container_engine() -> Option<&'static str> {
+    if which::which(DOCKER).is_ok() {
```

Comment:
I think this function can be simplified as `which::which(DOCKER).or_else(|| which::which(PODMAN))`.

Reply:
This would change the return type from `Option<&'static str>` to `Result<std::path::PathBuf, which::error::Error>`. Do we want to return the path to the command or just the command itself?

Reply:
Returning the path is fine.